### PR TITLE
[test] always showcase main media videos

### PR DESCRIPTION
**Pull request by akash1810**
## What does this change?

Testing if large main media video leads to more plays.
## What is the value of this and can you measure success?

⬆️ video plays.
## Does this affect other platforms - Amp, Apps, etc?

No.
## Screenshots

screen shot 2016-06-06 at 18 31 05
## Request for comment

@jamesgorrie @mr-mr

### CODE REVIEW

File: static/src/javascripts/bootstraps/enhanced/media/main.js [hunk 1]
```diff
@@ -407,6 +409,17 @@ define([
         }
     }
 
+    function showcaseMainMedia() {
+        $('.content__meta-container').addClass('content__meta-container--showcase');
```
Comment (akash1810) [inline static/src/javascripts/bootstraps/enhanced/media/main.js]: if test succeeds, we'll do this sever side

**Comment by jamesgorrie** [comment]
:

**Comment by jamesgorrie** [comment]
👍

**Comment by akash1810** [comment]
this'll go out next week when there's more audience availability

**Comment by akash1810** [comment]
updated with real audience numbers after baselining - ping @harrysalmon

**Comment by paperboyo** [comment]
Just wanted to mention that the weightings in Composer were thought as a Editorial tool to indicate the importance of (some!) content. So much so, that some are even behind a switch available only to some (designers). If Showcase for videos is not used like that, I would educate rather than just bluntly set all to that... Just my 2gr, obv!

**Comment by paperboyo** [comment]
OTOH, if the numbers from this test prove conclusive, there is going to be a good incentive to use video weightings more!

**Comment by akash1810** [comment]
> I would educate rather than just bluntly set all

Absolutely.

If the test proves positive, we'd likely default the weighting to showcase, but still allow the weighting to be changed. This is an assumption; lets wait for the test to complete before thinking too much on it 😄 .

**Comment by harrysalmon** [comment]
Looks good to me! We'll be able to detect anything above a 2% difference in the number of 25% video completes in 5 days time. This'll fill the AB test space too

**Pull request merged**